R/runMDS.R

```diff
@@ -11,23 +11,26 @@
 #' @param ... For the \code{calculateMDS} generic, additional arguments to pass to specific methods.
 #' For the SummarizedExperiment and SingleCellExperiment methods, additional arguments to pass to the ANY method.
 #'
-#' For \code{runMDS}, additional arguments to pass to \code{calculateMDS}. 
-#' @param method String specifying the type of distance to be computed between cells.
+#' For \code{runMDS}, additional arguments to pass to \code{calculateMDS}.
+#' @param FUN a function returning a \code{dist} object from a matrix, where rows are samples and rows are features
+#' @param keep_dist \code{TRUE} or \code{FALSE}: should the \code{dist} calculated by \code{FUN}
+#'   object be stored as \sQuote{dist} attribute of the matrix returned/stored by \code{calculateMDS}/
+#'   \code{runMDS}? (default: \code{keep_dist = FALSE})
```

Comment:
`Logical scalar indicating whether the \code{dist}`... No question mark either.

No need to specify the default, that's what the `usage` is for.